### PR TITLE
Fix ESBJAVA-5230

### PR DESCRIPTION
**Pull request by a5anka**
Read JMSType for content-type only if "transport.jms.ContentTypeProperty" or "transport.jms.ContentType" is not specified.

Refer - https://wso2.org/jira/browse/ESBJAVA-5230

### CODE REVIEW

File: components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/jms/JMSInjectHandler.java [hunk 1]
```diff
@@ -97,21 +96,26 @@ public boolean invoke(Object object, String name) throws SynapseException{
...
+
+            String contentTypeProperty =
+                    jmsProperties.getProperty(JMSConstants.CONTENT_TYPE_PROPERTY);
+            if (contentTypeProperty != null) {
+                contentType = msg.getStringProperty(contentTypeProperty);
             }
-            
-            if(contentType == null || contentType.trim().equals("")){
+
+            if (contentType == null || contentType.trim().equals("")) {
```
Comment (wggihan) [inline components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/jms/JMSInjectHandler.java]: Shall we use the isEmpty method in here?

File: components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/jms/JMSInjectHandler.java [hunk 2]
```diff
@@ -97,21 +96,26 @@ public boolean invoke(Object object, String name) throws SynapseException{
...
                 contentType = jmsProperties.getProperty(JMSConstants.CONTENT_TYPE);
             }
+
+            if (contentType == null) {
+                String jmsType = msg.getJMSType();
+
+                if (jmsType != null && !("".equals(jmsType.trim()))) {
```
Comment (wggihan) [inline components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/jms/JMSInjectHandler.java]: Again use isEmpty method

File: components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/jms/JMSInjectHandler.java (hunk 1 above)
Comment (a5anka) [reply components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/jms/JMSInjectHandler.java]: Fixed in b6ecdd3

File: components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/jms/JMSInjectHandler.java (hunk 2 above)
Comment (a5anka) [reply components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/jms/JMSInjectHandler.java]: Fixed in b6ecdd3

**Pull request merged**